src/lib/operators/import.cpp

```diff
@@ -51,9 +53,37 @@ std::shared_ptr<const Table> Import::_on_execute() {
   auto table = std::shared_ptr<Table>{};
 
   switch (_file_type) {
-    case FileType::Csv:
-      table = CsvParser::parse(filename, _chunk_size, _csv_meta);
+    case FileType::Csv: {
+      if (static_cast<int>(static_cast<bool>(_csv_meta)) +
+              static_cast<int>(Hyrise::get().storage_manager.has_table(_tablename)) +
+              static_cast<int>(std::filesystem::exists(filename + CsvMeta::META_FILE_EXTENSION)) >=
+          2) {
+        Fail(
+            "Cannot load table from csv. Table definition source is ambiguous. At least two of the following sources "
+            "gave CsvMeta information: Import; existing table in DBMS; Meta file next to .csv file");
+      }
+
+      CsvMeta csv_meta;
+      if (_csv_meta) {
```

Comment:
I just noticed that the `_csv_meta` optional of the `Import` operator is never set.
Since the `CsvParser` doesn't deal with such an optional anymore, let's get rid of it here, too.

Reply:
It does not seem to be implemented, as the LQP ImportNode does not have a property for this, you are right. @Bouncner is it fine if we completely remove the _csv_meta property from the Import operator? Or should it be possible to load the csv_meta from SQL at some point?

Reply:
Having CSV options is definitely needed as this PR is meant to allow us loading from TBL files which are not separated by commas.
So yes, we need it. But as we also could have a meta file with all parameters, it is not required.

Reply:
We might need clearer names here, as the csv meta file and `_csv_meta` are confusing.

Reply:
As there are a couple of comments here referring to this topic: did we completely settle it last week? If not, let me know.

If so, feel free to resolve all the comments.